README.md

```diff
@@ -69,12 +69,25 @@ If you want to install HDM using RVM, you need the following modules:
 
 ### Beginning with betadots HDM
 
-The most simple approach is to just include the hdm class:
+The most simple approach is to just include the hdm class and provide the version parameter:
 
 ```puppet
 include hdm
 ```
 
+You can find the released versions on [HDM releases page](https://github.com/betadots/hdm/releases)
+```yaml
+hdm::version: '1.0.1
+```
+
+Another option is to use class resource type declaration:
+
+```puppet
+class { 'hdm':
+  version => '1.0.1',
+}
+```
+
```

Comment:
I would drop Hiera stuff completely:

    The most simple approach is to just include the hdm class and provide the version parameter:
    
    ```puppet
    class { 'hdm':
      version => '1.0.1',
    }
    ```
    
    You can find the released versions on [HDM releases page](https://github.com/betadots/hdm/releases)

People who are used to to this old-style pattern with Hiera know how to "convert", and those who don't won't be confused ("include" in the sentense is probably not the best, but i am fine with it and can't figure out something concise and suited).

Reply:
Ah, lol, it's a hiera data visualization tool :facepalm:. Maybe it make sense in that case :shrug: